database/database.go

```diff
@@ -0,0 +1,104 @@
+package database
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/josephroberts/edge-node-manager/config"
+
+	"log"
+)
+
+func Start() error {
+	err := exec.Command(os.Getenv("GOPATH")+"/bin/tiedot", "-mode=httpd", "-dir="+filepath.Join(config.Persistant, config.Db), "-port="+config.DbPort).Start()
+	if err != nil {
+		log.Println("Failed to start database")
+		return err
+	}
+
+	time.Sleep(1 * time.Second)
+
+	return initialise()
+}
+
+func initialise() error {
+	resp, err := http.Get("http://localhost:" + config.DbPort + "/create?col=Devices")
```

Comment:
More, generally you're repeating this block of code 3 times unnecessarily. It should be separated into a helper function, regardless, e.g. something like:

```go
func getLocalUri(path string) (string, error) {
	uri := fmt.Sprintf("http://localhost:%s/%path", config.DbPort, path)

	resp, err := http.Get(uri)

	if err != nil {
		return "", err
	}

	bytes, err := ioutil.ReadAll(resp.Body)
	resp.Body.Close()

	if err == nil {
		return string(bytes), nil
	}

	return "", err
}
```

Then you could do:

```go
    if _, err := getLocalUrl("create?col=Devices"); err != nil {
        log.Println("Failed to create Devices collection")
        return err
    }
    ...
```

The `resp.Body.Close()` is something a helper function should definitely be taking care of.